assets/stylesheets/components/_avatar-group.scss

```diff
@@ -0,0 +1,22 @@
+$base-size: 16px;
```

Comment:
Seems like we should be using the global `$base-font-size` variable instead of redeclaring this value here... 

https://github.com/crdschurch/crds-styles/blob/development/assets/stylesheets/_variables.scss#L6